cluster/images/hyperkube/Makefile

```diff
@@ -19,7 +19,7 @@
 
 REGISTRY?=gcr.io/google_containers
 ARCH?=amd64
-TEMP_DIR:=$(shell mktemp -d)
+TEMP_DIR=/tmp/hyperkube
```

Comment:
it will be removed and (re)created, if needed.  The idea is that you have a well known location for the build directory. So you can check and debug the intermediate results easily. 

Also, I was thinking about
*rename TEMP_DIR to BUILD_DIR. 
*use another location .e.g. ./_output or ../../../_output/